maven-resolver/src/main/java/org/wildfly/channel/maven/ChannelCoordinate.java

```diff
@@ -44,4 +44,15 @@ public ChannelCoordinate(String groupId, String artifactId) {
     public ChannelCoordinate(URL url) {
         super(url);
     }
+    
+    @Override
+    public String getClassifier() {
```

Comment:
In the case of the wildfly-maven-plugin, the default constructor is set. So no default values for classifier and extension are set.
We need fixed classifier and extension. Otherwise they are set to null.

Reply:
OK I see